src/cli/cli.cpp

```diff
@@ -1228,6 +1229,39 @@ otError Interpreter::ProcessCounters(Arg aArgs[])
             ExitNow(error = OT_ERROR_INVALID_ARGS);
         }
     }
+    else if(aArgs[0]=="ip")
+    {
+        if (aArgsLength == 1)
```

Comment:
@arun-mahasenan can you please rebase and update the code? 
- The `aArgLength` param is no longer available or used since https://github.com/openthread/openthread/pull/6767)